src/components/pill/pill.spec.tsx

```diff
@@ -589,6 +612,85 @@ describe("Pill", () => {
               });
             });
           });
+
+          describe("when pillRole is status and isDarkbackground prop is set to true", () => {
```

Comment:
comment(non-blocking): we'd like to avoid nested describe blocks as much as possible in the future

Don't change this now - we will likely address this when we do our refactors over to RTL. Just mentioning it as something to be aware of